exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/SQLServerDialect.kt

```diff
@@ -118,6 +118,22 @@ internal object SQLServerFunctionProvider : FunctionProvider() {
         append("DATEPART(MINUTE, ", expr, ")")
     }
 
+    override fun <T> stdDevPop(expression: Expression<T>, queryBuilder: QueryBuilder): Unit = queryBuilder {
+        append("STDEVP(", expression, ")")
+    }
+
+    override fun <T> stdDevSamp(expression: Expression<T>, queryBuilder: QueryBuilder): Unit = queryBuilder {
+        append("STDEV(", expression, ")")
```

Comment:
Could you check if `STDEV for this case is returning the same value as `STDDEV_SAMP` in the other DB? It should be the different functions

Reply:
SQL Server's [documentation](https://learn.microsoft.com/en-us/sql/t-sql/functions/stdev-transact-sql?view=sql-server-ver16) for `STDEV` does use a different term than `STDDEV_SAMP` in [MySQL](https://dev.mysql.com/doc/refman/8.0/en/aggregate-functions.html#function_stddev-samp) or [PostgreSQL](https://www.postgresql.org/docs/15/functions-aggregate.html#FUNCTIONS-AGGREGATE-STATISTICS-TABLE), for example:

> statistical standard deviation of all values

vs

> sample standard deviation of input values

But I ran current tests, as well as another with a larger sample size, and confirmed that the value returned by SQL Server's `STDEV` is equivalent to the values returned from other DBs `STDDEV_SAMP`.